src/api/serviceConfig.ts

```diff
@@ -0,0 +1,8 @@
+const serviceConfig: any = {
+  dnsGd: {
```

Comment:
no its `dns_gd`. Please refer to this doc: https://github.com/Neilpang/acme.sh/wiki/dnsapi#4-use-godaddycom-domain-api-to-automatically-issue-cert